tables/de-accents-detailed.cti

```diff
@@ -1,6 +1,7 @@
 # liblouis: Letters with accents (without ä ö ü) - detailed representation of french and italian accents
 #
 #  Copyright (C) 2018 SBS Schweizerische Bibliothek für Blinde, Seh- und Lesebehinderte
+#  Copyright (C) 2020 Bue Vester-Andersen
```

Comment:
A copyright line is not really appropriate in case of an addition of a single comment.

Reply:
> Should there be anything else to indicate that I made changes to the file?

No, that's not needed.